runtime/lib/component/light.js

```diff
@@ -179,10 +182,13 @@ Light.prototype.lightMethod = function (name, args) {
  * @param {boolean} dndMode true if opened
  */
 Light.prototype.setDNDMode = function (dndMode) {
+  var caps = new Caps()
   if (dndMode) {
-    this.lightMethod('setGlobalAlphaFactor', [DND_MODE_ALPHA_FACTOR])
+    caps.writeDouble(DND_MODE_ALPHA_FACTOR)
+    this.runtime.component.flora.post('rokid.lightd.global_alpha_factor', caps, Flora.MSGTYPE_PERSIST)
```

Comment:
We do not need to use cap explicitly now, just use [] instead.

Reply:
OK, updated